{
  "comments": [
    {
      "key": {
        "uuid": "5ac10518_4f1424f5",
        "filename": "dovetail/run.py",
        "patchSetId": 6
      },
      "lineNbr": 41,
      "author": {
        "id": 5469
      },
      "writtenOn": "2016-11-07T08:03:13Z",
      "side": 1,
      "message": "Maybe this can be logger.error to emphasize this problem.",
      "range": {
        "startLine": 41,
        "startChar": 12,
        "endLine": 41,
        "endChar": 23
      },
      "revId": "34c1d09e90f5d35a10fb92288b95e00148047714",
      "serverId": "bbac25d2-bf60-4904-9ba8-a72fc000d6c5",
      "unresolved": false
    }
  ]
}